packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -213,25 +210,23 @@ export class VerticalStackedBarChartBase extends React.Component<
   };
 
   private _getFormattedLineData = (data: IVerticalStackedChartProps[]): LineObject => {
-    const linesData: LineData[] = [];
+    const linesData: LinePoint[] = [];
     const formattedLineData: LineObject = {};
     data.forEach((item: IVerticalStackedChartProps, index: number) => {
       if (item.lineData) {
-        linesData.push(...item.lineData);
         // injecting corresponding x data point in each of the line data
         // we inject index also , it will be helpful to draw lines when x asix is
         // of string type
-        linesData.forEach((line: LineData) => {
-          if (line.x === undefined) {
-            line.x = item.xAxisPoint;
-          }
-          if (line.index === undefined) {
-            line.index = index;
-          }
+        item.lineData.forEach(line => {
+          linesData.push({
+            ...line,
+            index,
+            x: item,
```

Comment:
@mbest  why are you setting `item` to `x` instead of `x.xAxisPoint`

Reply:
Using `xAxisPoint` means having to do a `find` later. This change directly links to the item. I've renamed the property to make it a bit more informative.